### ISSUE TITLE
When analyzing a macOS 13 disk image with AUTOSTART plugin, the results cannot be written to mac_apt.db

### ISSUE THREAD

**Issue by mnrkbys** [ydkhatri/mac_apt#96]
I am testing to analyze macOS 13 disk images.
However, the following error is occurred at the end of execution AUTOSTART plugin.
I have confirmed the error when analyzing macOS 13.2.1 and macOS 13.3.1(a).

```
2023-05-09 12:13:43|MAIN.AUTOSTART|INFO|Found 1359 autostart item(s)
2023-05-09 12:13:43|MAIN.HELPERS.WRITER|DEBUG|Trying to write out autostart information
2023-05-09 12:13:43|MAIN.HELPERS.WRITER|ERROR|Error binding parameter 4 - probably unsupported type.
2023-05-09 12:13:43|MAIN.HELPERS.WRITER|ERROR|AutoStart
Traceback (most recent call last):
  File "/Users/macforensics/Documents/GitHub/mac_apt/plugins/helpers/writer.py", line 398, in WriteRows
    cursor.executemany(query, rows)
sqlite3.InterfaceError: Error binding parameter 4 - probably unsupported type.
```

The log files are attached below.
Do you have any ideas for a solution?

macOS 13.2.1
[Log.20230509-121213.txt](https://github.com/ydkhatri/mac_apt/files/11426861/Log.20230509-121213.txt)

macOS 13.3.1(a)
[Log.20230509-120601.txt](https://github.com/ydkhatri/mac_apt/files/11426840/Log.20230509-120601.txt)

**Comment by mnrkbys** [comment]
Ok, I found the root cause of this issue.
Two plist files in /System/Library/LaunchDaemons/, com.apple.containermanagerd.plist and com.apple.containermanagerd.system.plist have a "Disabled" key, but its type is dict, not bool.
So, AUTOSTART plugin will be crashed when the results are written to mac_apt.db.

I have made a patch for this and will register a pull request later.

```
% plutil -p /System/Library/LaunchDaemons/com.apple.containermanagerd.plist
{
  "Disabled" => {
    "#IfFeatureFlagEnabled" => "containermanagerd/user_system_split"
    "#Then" => 1
  }
  "EnablePressuredExit" => 1
  "EnableTransactions" => 1
  "Label" => "com.apple.containermanagerd"
  "MachServices" => {
    "com.apple.containermanagerd" => 1
  }
  "ProcessType" => "Adaptive"
  "ProgramArguments" => [
    0 => "/usr/libexec/containermanagerd"
    1 => "--runmode=privileged"
    2 => "--user-container-mode=current"
    3 => "--bundle-container-mode=global"
    4 => "--bundle-container-owner=_appinstalld"
    5 => "--system-container-mode=none"
  ]
}

% plutil -p /System/Library/LaunchDaemons/com.apple.containermanagerd.system.plist 
{
  "Disabled" => {
    "#IfFeatureFlagDisabled" => "containermanagerd/user_system_split"
    "#Then" => 1
  }
  "EnablePressuredExit" => 1
  "EnableTransactions" => 1
  "Label" => "com.apple.containermanagerd.system"
  "MachServices" => {
    "com.apple.containermanagerd" => 1
    "com.apple.containermanagerd.system" => 1
  }
  "ProcessType" => "Adaptive"
  "ProgramArguments" => [
    0 => "/usr/libexec/containermanagerd_system"
    1 => "--runmode=privileged"
    2 => "--user-container-mode=current"
    3 => "--bundle-container-mode=global"
    4 => "--bundle-container-owner=_appinstalld"
    5 => "--system-container-mode=none"
  ]
}
```

**Comment by mnrkbys** [comment]
This issue has been resolved by #97.
